test/e2e/federation-apiserver.go

```diff
@@ -34,52 +34,54 @@ import (
 var _ = framework.KubeDescribe("Federation apiserver [Feature:Federation]", func() {
 	f := framework.NewDefaultFederatedFramework("federation-cluster")
 
-	AfterEach(func() {
-		// framework.SkipUnlessFederated(f.Client)  TODO: quinton-hoole: Temporarily disabled
-
-		// Delete registered clusters.
-		// This is if a test failed, it should not affect other tests.
-		clusterList, err := f.FederationClientset.Federation().Clusters().List(api.ListOptions{})
-		Expect(err).NotTo(HaveOccurred())
-		for _, cluster := range clusterList.Items {
-			err := f.FederationClientset.Federation().Clusters().Delete(cluster.Name, &api.DeleteOptions{})
+	Describe("testing clusters", func() {
```

Comment:
nit: Reword this to be "Cluster objects" and the It() clause to "should be created and destroyed successfully".